src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -114,10 +116,14 @@ public function dump(array $options = array())
             'namespace' => '',
             'as_files' => false,
             'debug' => true,
+            'hot_path_tag' => null,
```

Comment:
Why does it not have a default?

Reply:
I don't want it to be enabled by default, so this is opt-in.
This is the way to do it.